pkgs/top-level/all-packages.nix

```diff
@@ -24092,11 +24092,6 @@ in
 
   kile = libsForQt5.callPackage ../applications/editors/kile { };
 
-  kino = callPackage ../applications/video/kino {
```

Comment:
You seem to have forgotten to actually delete this folder?

Reply:
indeed